### PR TITLE
Fix cursor scaling on wayland

### PR DESCRIPTION
**Pull request by xduugu**
See https://github.com/electron/electron/issues/19810

### CODE REVIEW

File: com.github.IsmaelMartinez.teams_for_linux.yml [hunk 1]
```diff
@@ -22,6 +22,8 @@ finish-args:
   - --talk-name=org.gtk.Notifications
   - --talk-name=org.gnome.SessionManager
   - --env=XDG_CURRENT_DESKTOP=Unity
+  # required to fix cursor scaling on wayland
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
```
Comment (jijojosephk) [inline com.github.IsmaelMartinez.teams_for_linux.yml:26]: @bbhtt would you mind reviewing this one?

Comment (bbhtt) [reply com.github.IsmaelMartinez.teams_for_linux.yml:26]: Yes, LGTM this is a known issue, see https://github.com/flathub/org.electronjs.Electron2.BaseApp/issues/42

**Review by bbhtt — APPROVED** [review]

File: com.github.IsmaelMartinez.teams_for_linux.yml (hunk 1 above)
Comment (jijojosephk) [reply com.github.IsmaelMartinez.teams_for_linux.yml:26]: Thanks!

**Comment by jijojosephk** [comment]
Thanks @xduugu and @bbhtt :rocket:

**Comment by jijojosephk** [comment]
Please join this chat room if you're interested. 

https://matrix.to/#/#teams-for-linux_community:gitter.im

**Pull request merged**